### ISSUE TITLE
Model comparison with FiVE

### ISSUE THREAD

**Issue by NBitBuilder** [Jiangbo-Shi/ViLa-MIL#3]
Hi,

Thank you for your contributions. Your idea for a vision-language pathology model is quite interesting.

However, I noticed that there is another paper presented at CVPR 2024 on a slide-level vision-language model, which demonstrates significantly better performance compared to your proposed results.

`https://github.com/ls1rius/WSI_FiVE
`
For example, on the same TCGA Lung classification task, the WSI_FiVE model achieved an accuracy of **91.25%** with a **16-shot** learning setup, whereas ViLa achieved only **67.7%.** This is a remarkable difference!

Could you please provide more insights into the advantages and disadvantages of your approach, and explain what might be causing this significant performance gap?

**Comment by NBitBuilder** [comment]
Here are the results from WSI_FiVE

![image](https://github.com/Jiangbo-Shi/ViLa-MIL/assets/61983243/6a62fe5e-b929-4fe0-8fc9-61da7bde048b)

**Comment by Jiangbo-Shi** [comment]
Thanks for your question. 
FiVE is an interesting and awesome work that utilizes paired pathological reports as the semantic supervision to pre-train the model. The reasons for the performance difference between our ViLa-MIL and FiVE are due to the varied experiment settings. Here we provide a simple comparison of three aspects:
1.  Supervision: Our ViLa-MIL only relies on the few-shot class labels for training with the help of LLM-generated textual descriptive prompts. FiVE needs to introduce extra paired pathological reports as guidance. 
2. Model pertain: The motivation behind our ViLa-MIL is to utilize the great multi-modal alignment ability of current popular VLMs to boost the performance of the MIL framework in more challenging settings. Both image and text encoders are frozen and only two lightable decoders and defined prompts are needed to fine-tune, which is very parameter-efficient. FiVE can be treated as a pre-training framework, while the text encoder is still needed to pre-train. These two works utilize different paradigms. 
3. Data preprocessing: The data preprocessing and dataset split settings between these two works are also different. So directly comparing the experimental results might be inappropriate.

**Comment by NBitBuilder** [comment]
Thank you for getting back to me. I will check the implementations. Regardless of your explanations, the ACC with more than 20% is astonishing. One of the models could have some implicit defects, perhaps.
